openshift/app.bc.yaml

```diff
@@ -45,6 +45,7 @@ objects:
           FROM BuildConfig
           # Install python dependency for node-gyp rebuild
           RUN apk update && apk add python make g++
+          ENV NO_UPDATE_NOTIFIER=true
```

Comment:
ENV declaration lines need to be done before any run statements by convention. While this doesn't actually affect anything here in this case as python apk install isn't dependent on that ENV variable, it should still be declared on Line 46 right after the FROM declaration.

Reply:
yeah, was just worried about mucking up python install. 
moved it.